parlai/agents/bert_ranker/bi_encoder_ranker.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, opt, shared=None):
 
     def build_model(self):
         self.model = BiEncoderModule(self.opt)
+        if self.use_cuda:
```

Comment:
I think the goal should be to get rid of this required boilerplate in build_model